src/main/java/org/thingsboard/gateway/service/gateway/MqttGatewayService.java

```diff
@@ -657,8 +656,8 @@ private MqttClient initMqttClient() {
 
             tbClient.on(DEVICE_ATTRIBUTES_TOPIC, this).await(connection.getConnectionTimeout(), TimeUnit.MILLISECONDS);
             tbClient.on(DEVICE_GET_ATTRIBUTES_RESPONSE_PLUS_TOPIC, this).await(connection.getConnectionTimeout(), TimeUnit.MILLISECONDS);
-            tbClient.on(DEVICE_GET_ATTRIBUTES_RESPONSE_PLUS_TOPIC, this).await(connection.getConnectionTimeout(), TimeUnit.MILLISECONDS);
 
+            tbClient.on(GATEWAY_RESPONSES_ATTRIBUTES_TOPIC, this).await(connection.getConnectionTimeout(), TimeUnit.MILLISECONDS);
```

Comment:
Why is this space needed?

Reply:
659 blank lines are used to differentiate gateway topic and device topic.
If you don't need it, I can delete it.